sdk-core/src/main/java/care/data4life/sdk/network/model/VersionList.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2020 D4L data4life gGmbH / All rights reserved.
+ *
+ * D4L owns all legal rights, title and interest in and to the Software Development Kit ("SDK"),
+ * including any intellectual property rights that subsist in the SDK.
+ *
+ * The SDK and its documentation may be accessed and used for viewing/review purposes only.
+ * Any usage of the SDK for other purposes, including usage for the development of
+ * applications/third-party applications shall require the conclusion of a license agreement
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+package care.data4life.sdk.network.model
+
+import com.squareup.moshi.Json
+
+data class VersionList(
+        @field:Json(name = "versions")
+        override val versions: List<Version>
+) : NetworkModelContract.VersionList {
+
+
+    fun isSupported(currentVersion: String): Boolean {
+        var isSupported = true
+        for (version in versions) {
+            if (version.name == currentVersion.substring(0,5) && version.status == NetworkModelContract.Version.KEY_UNSUPPORTED) {
```

Comment:
What is that supposed to do?

Reply:
getVersionInfo should be changed to what?

Reply:
No the `substring(x,y)` part. (sry, I forget always does not mark single parts in a line)

Reply:
If the version exit and has status UNSUPPORTED: then we don't support this version and must fails the login.

Reply:
True, but why we need the a substring of the version for that? (That's the bit I do not get)
So maybe make an example, which that should cover.

Reply:
I added it to run under debug as well. Because the build name = 1.9.0-config-debug-something so this version will not be found unless you trim it to 1.9.0

Reply:
To be honest, I am a bit uncertain about this, for some reasons. However if you wanna cover such case I guess you should not use substring since it has a fixed range. So for example `1.9.32` would be cut to `1.9.3`, which it should not. You can use a regex for that or split `-` with the assumption that it will be the only delimiter we ever use.

Reply:
Also it needs to be covered by a test as well.

Reply:
The test is added already. I thought about that but I don't think we support 1.9.32..the third digits is for fixing bugs, the second for features.

Reply:
I mean, I can make another example - the upcoming `1.10.0`, which will have a similar issue. Also I cannot spot any test, which tests this trimming behaviour. Did you pushed the test?

Reply:
not a specific test just set the example lijke the debug version.